control/tests/docstrings_test.py

```diff
@@ -30,9 +30,9 @@
 function_docstring_hash = {
     control.append:                     '1bddbac0fe932755c85e9fb0bfb97d88',
     control.describing_function_plot:   '95f894706b1d3eeb3b854934596af09f',
-    control.dlqe:                       '9db995ed95c2214ce97074b0616a3191',
-    control.dlqr:                       '896cfa651dbbd80e417635904d13c9d6',
-    control.lqe:                        '567bf657538935173f2e50700ba87168',
+    control.dlqe:                       'f2e52e35692cf5ffe911684d41d284c9',
```

Comment:
I think these hashes are new-ish, just got caught out by them working on adding ruff-pyflakes checks.  Is the idea to require the developer changing code to consider implications on the docstring?  Or, I suppose, vice versa.

Reply:
Yes, that was the idea.  However, they will go away when #1094 is merged (I found a better way to handle them).